app/models/doughnut.rb

```diff
@@ -3,4 +3,5 @@ class Doughnut < ApplicationRecord
     validates :price, presence: true
     validates :quantity, presence: true
     validates :description, presence: true
+    has_many :orders
```

Comment:
Isn't it the other way around? An order has one or more doughnuts?

Reply:
yeah it's confusing I know, but basically our DB schema wants each Order entity to have a doughnut_id foreign key column to reflect which doughnut was bought in that order. For now, each order will only consist of exactly one doughnut, and logically yes, it should be the other way around. But Rails is weird so if we use, `has_one :doughnut`, Rails will instead add a foreign_key `order_id` to each Doughnut entity, which is not what we want. `has_many :orders` will allow us to query all orders for a specific Doughnut entity here.

Reply:
Ah gotcha okay I understood it the other way but that makes sense.